cmd/oci-image-tool/validate.go

```diff
@@ -67,6 +68,11 @@ func newValidateCmd(stdout, stderr *log.Logger) *cobra.Command {
 		`A set of refs pointing to the manifests to be validated. Each reference must be present in the "refs" subdirectory of the image. Only applicable if type is image or imageLayout.`,
 	)
 
+	cmd.Flags().BoolVarP(
+		&v.verbose, "verbose", "v", false,
+		fmt.Sprintf(`Output verbosely, including all objects digest to be validated. If unset, oci-image-tool will only output a OK or failed result.`),
+	)
```

Comment:
@wking I'm sure you can find exceptions to my generalizations. :)

For our purposes, I would expect log level changes to provide more information about what is happening, while an increase in verbosity would provide richer test output.